tasks/main.yml

```diff
@@ -103,34 +97,6 @@
       loop_control:
         loop_var: user
         label: "{{ user.name }} {{ user.state }}"
-      when:
-        # Ansible user module and usermod command fail to modify any logged in user when uid is set
-        # Workaround is to run usermod manually for logged in users and not set UID even if it is defined
-        - user.uid is not defined or (user.uid is defined and user.name not in admin_logged_in_users)
-
-    - name: command usermod - primary group and extra groups, workaround for logged in users
-      command: usermod -g {{ user.group }} -G {{ user.groups }} {{ user.name }}
-      with_items: "{{ admin_list_of_user_lists | default([]) }}"
-      loop_control:
-        loop_var: user
-        label: "{{ user.name }}"
-      when:
-       - user.name in admin_logged_in_users
-       - user.group is defined
-       - user.groups is defined
-       - user.uid is defined # We only need this usermod command to set group and groups for any logged in users
-
-    # An extra exception if the group key is not defined for the logged in user
-    - name: command usermod - extra groups, workaround for logged in user
```

Comment:
I don't think this is necessary in the recent ansible since the users task was able to be even if the user was logged in.